### PR TITLE
Add "active" reviewers filter

### PR DESCRIPTION
**Pull request by zerolab**

### CODE REVIEW

**Comment by zerolab** [comment]
OK, this should be ready. Let's discuss

File: opentech/apply/review/models.py [hunk 1]
```diff
@@ -72,3 +74,25 @@ def __str__(self):
...
         return f'<{self.__class__.__name__}: {str(self.review)}>'
+
+
+@receiver(post_save, sender=Review)
+def update_submission_reviewers_list(sender, **kwargs):
+    review = kwargs.get('instance')
+
+    if not review.submission.reviewers.filter(id=review.author.id).exists():
+        review.submission.reviewers.add(review.author)
+        review.submission.save()
```
Comment (todd-dembrey) [inline opentech/apply/review/models.py]: You dont need to save after an m2m add. Unless there is something else you are trying to trigger?

File: opentech/apply/review/models.py [hunk 2]
```diff
@@ -72,3 +74,25 @@ def __str__(self):
...
+
+        Activity.actions.create(
+            user=review.author,
+            submission=review.submission,
+            message=f'Added themselves to the Reviewers list.'
+        )
```
Comment (todd-dembrey) [inline opentech/apply/review/models.py]: I don't think we want to track this. It feel like it should be an implicit part of submitting a review if they aren't in the list.

`Dan added himself to reviewers`
`Dan submitted a review`

File: opentech/apply/review/models.py [hunk 3]
```diff
@@ -72,3 +74,25 @@ def __str__(self):
...
+
+    if not review.id:
```
Comment (todd-dembrey) [inline opentech/apply/review/models.py]: Dont you want the created Boolean from kwargs?

**Pull request merged**